vmdb/db/fixtures/ae_datastore/ManageIQ/Cloud/VM/Provisioning/Placement.class/__methods__/best_fit_amazon.rb

```diff
@@ -15,3 +15,41 @@
 #  best Availability Zone for you based on system health and available capacity. If you launch additional
 #  instances, only specify an Availability Zone if your new instances must be close to, or separated from,
 #  your running instances."
+#
+#
+# VPC & Subnet
+# For C4 and T2 instances we have to provide VPC and Subnet.
+# http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/t2-instances.html
```

Comment:
I ever knew about this page...would be really good to update the MiqEc2InstanceTypes.rb with a link to this (and then I can also monitor changes on this page for the future if they change anything).